app/controllers/concerns/sms.rb

```diff
@@ -2,36 +2,16 @@
 require "twilio-ruby"
 
 # A set of utility methods used in relation to SMS notifications
-# @author Eric Schlange <eric.schlange@northwestern.edu>
 module Sms
-  extend ActiveSupport::Concern
-
-  # Send an SMS message body to a specified receiving participant (leaving
-  # off recipient defaults the value to the current participant)
-  # rubocop:disable Metrics/AbcSize
-  def send_sms(recipient = current_user, message_body)
+  def send_sms(recipient, message_body)
+    return unless Rails.env.staging? || Rails.env.production?
```

Comment:
Small thing: not sure if we want to allow for too many exit paths for a ruby method. We can combine the logic for the exit logic like so:
```return unless (Rails.env.staging? || Rails.env.production? ) && !recipient.phone_number.blank?```

@ericcf @michaelwehrley I'd like to propose that we utilize an augmented SESE principle. With Ruby code we should have SEDE (single entry, double exit). The first exit being one that wraps a code bock with an unless statement, the second being  the implicit return of the ruby method (the final statement.)

It is partly selfish and partly not: I've been writing code this way my entire life and there is some decent research and metrics gathered about how code is less complex and more readable when using SESE (http://codebetter.com/patricksmacchia/2008/03/07/a-simple-trick-to-code-better-and-to-increase-testability/). 

Please let me know your thoughts.